eland/field_mappings.py

```diff
@@ -100,6 +100,8 @@ def is_es_agg_compatible(self, es_agg) -> bool:
             elif es_agg[0] == "percentiles":
                 es_agg = "percentiles"
 
+        if es_agg == "value_count":
```

Comment:
Can move this to `es_agg == "cardinality"` -> `es_agg in ("cardinality", "value_count")`

Also should update the comment "Cardinality works for all types" -> "Cardinality and Count work for all types"

Reply:
Yep